operator/pkg/controllers/master/kubeapiserver.go

```diff
@@ -455,6 +455,10 @@ var (
 	disabledFlagsForAPIServer = map[string]struct{}{"--feature-gates": {}}
 )
 
+var (
+	disabledFlagsForApi126 = map[string]struct{}{"--feature-gates": {}, "--logtostderr": {}}
```

Comment:
Comment should be added for why each flag is disabled.
For `--feature-gates` flag, only `TTLAfterFinished`is related. You show call that out.